pinot-ingestion-jobs/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
if we don't have this pom, then we need to set modules in root pom.xml:
```
    <module>pinot-ingestion-jobs/pinot-ingestion-common</module>
    <module>pinot-ingestion-jobs/pinot-hadoop</module>
    <module>pinot-ingestion-jobs/pinot-spark</module>
```
instead of 
```
    <module>pinot-ingestion-jobs</module>
```